build.gradle

```diff
@@ -133,13 +133,13 @@ subprojects {
             check 'UnsynchronizedOverridesSynchronized', CheckSeverity.ERROR
             check 'WaitNotInLoop', CheckSeverity.ERROR
             disable 'UnusedVariable' // Workaround for https://github.com/google/error-prone/issues/1250
-        }
-        options.incremental = true
 
-        // workaround for https://github.com/gradle/gradle/issues/2510
-        if (JavaVersion.current() >= JavaVersion.VERSION_1_9) {
-            options.compilerArgs += ['--release', '8']
+            // Finally check is broken in Java 12.  See https://github.com/google/error-prone/issues/1257
+            if ((JavaVersion.current().majorVersion as Integer) > 11) {
```

Comment:
Is there a specific reason why `JavaVersion.current() => JavaVersion.VERSION_12` isn't used here?